client/my-sites/site-settings/spam-filtering-settings/index.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormToggle from 'components/forms/form-toggle';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import SectionHeader from 'components/section-header';
+import { isJetpackModuleActive } from 'state/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { updateSettings } from 'state/jetpack/settings/actions';
+
+const SpamFilteringSettings = ( {
+	fields,
+	handleAutosavingToggle,
+	isRequestingSettings,
+	isSavingSettings,
+	akismetActive,
+	translate
+} ) => {
+	return (
+		<div>
+			<SectionHeader label={ translate( 'AntiSpam' ) } />
+			<Card className="spam-filtering-settings__card site-settings site-settings__module-settings">
+				<FormFieldset className="spam-filtering-settings__formfieldset">
+					<div className="spam-filtering-settings__info-link-container site-settings__info-link-container">
+						<InfoPopover position={ 'left' }>
+							<ExternalLink target="_blank" icon={ true } href={ 'https://jetpack.com/features/security/spam-filtering/' } >
+								{ translate( 'Learn more about Spam Filtering' ) }
+							</ExternalLink>
+						</InfoPopover>
+					</div>
+					<div className="spam-filtering-settings__module-settings">
+						<FormToggle
+							className="is-compact"
+							checked={ fields.akismet_show_user_comments_approved || false }
+							disabled={ isRequestingSettings || isSavingSettings || ! akismetActive }
+							onChange={ handleAutosavingToggle( 'akismet_show_user_comments_approved' ) } >
+							{ translate( 'Show the number of approved comments beside each comment author.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Show the number of approved comments beside each comment author' )` *ES Score: 16*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/akismet-card-for-jetpack-sites-in-settings-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).